fe/fe-core/src/main/java/com/starrocks/connector/elasticsearch/EsRestClient.java

```diff
@@ -280,8 +294,92 @@ private static SSLSocketFactory createSSLSocketFactory() {
             sc.init(null, new TrustManager[] {new TrustAllCerts()}, new SecureRandom());
             ssfFactory = sc.getSocketFactory();
         } catch (Exception e) {
-            throw new StarRocksESException("Errors happens when create ssl socket");
+            throw new StarRocksConnectorException("Errors happens when create ssl socket");
         }
         return ssfFactory;
     }
+
+    @Data
+    public static class EsIndex {
+        private String index;
+    }
+
+    /**
+     * response
+     [
+        {
+        "index": ".kibana_1"
+        },
+        {
+        "index": ".opendistro_security"
+        },
+        {
+        "index": "kibana_sample_data_ecommerce"
+        },
+        {
+        "index": "kibana_sample_data_ecommerce_2"
+        },
+        {
+        "index": "kibana_sample_data_flights"
+        },
+        {
+        "index": "kibana_sample_data_logs"
+        }
+     ]
+     * indices are same as table
+     * @return
+     */
+    private Set<String> getIndices() {
+        String response = execute("_cat/indices?h=index&format=json&s=index:asc");
+        if (Objects.isNull(response)) {
+            throw new StarRocksConnectorException("es indexes are null, maybe this is error");
+        }
+
+        EsIndex[] esIndices = getFromJSONArray(response, EsIndex[].class);
+        return Arrays.asList(esIndices).stream()
+                .filter(index -> !index.getIndex().startsWith("."))
+                .map(index -> index.getIndex())
+                .collect(Collectors.toSet());
+    }
+    /**
+     {
+        "kibana_sample_data_ecommerce": {
+            "aliases": {}
+        }
+     }
+     * Get all alias.
+     **/
+    private Map<String, List<String>> getAliases() {
+        String response = execute("_aliases");
+        // key: index, value: aliases
+        Map<String, List<String>> aliases = new HashMap<>();
+        JsonNode root = readTree(response);
+        if (root == null) {
+            return aliases;
+        }
+        Iterator<Map.Entry<String, JsonNode>> indexElements = root.fields();
+        while (indexElements.hasNext()) {
+            Map.Entry<String, JsonNode> indexAlias = indexElements.next();
+            JsonNode aliasesNode = indexAlias.getValue().get("aliases");
+            Iterator<String> aliasNames = aliasesNode.fieldNames();
+            if (aliasNames.hasNext()) {
+                aliases.put(indexAlias.getKey(), ImmutableList.copyOf(aliasNames));
+            }
+        }
+        return aliases;
+    }
+
+    /**
+     * get tables union between indices and aliases
+     * @return
+     */
+    public List<String> listTables() {
+        Set<String> indices = getIndices();
+        Map<String, List<String>> aliases = getAliases();
+        aliases.entrySet().stream()
+                .filter(e -> (!e.getKey().startsWith(".") && !indices.contains(e.getKey())))
```

Comment:
Can we do filter `startsWith(".")` in `getAliases()`, just the same as `getIndices()`